src/extension/Content/index.ts

```diff
@@ -1,28 +1,11 @@
 import { LiveSelector, MutationObserverWatcher } from '@dimensiondev/holoflows-kit'
 import browser from 'webextension-polyfill'
-
-import { TopNavMenuSelector, VideoFeedSelector } from './Selectors/twitter'
-
-// add a link on nav menu
-const navWrapper = new LiveSelector().querySelector<HTMLDivElement>(TopNavMenuSelector).enableSingleMode().evaluate()
-
-if (navWrapper) {
```

Comment:
@c3V6a2Vy what is the reason to delete these web injection for tiktok?

Reply:
Oh I thought the top nav bar were for twitter? Are they not?